mmdet/models/dense_heads/rpn_head.py

```diff
@@ -79,14 +79,14 @@ def loss(self,
         return dict(
             loss_rpn_cls=losses['loss_cls'], loss_rpn_bbox=losses['loss_bbox'])
 
-    def _get_bboxes_single(self,
-                           cls_scores,
-                           bbox_preds,
-                           mlvl_anchors,
-                           img_shape,
-                           scale_factor,
-                           cfg,
-                           rescale=False):
+    def _get_bboxes(self,
```

Comment:
1 comments the length of the list
2 (num_points * num_classes, H, W) in docstr maybe wrong?